stylelint.config.js

```diff
@@ -1,6 +1,3 @@
 export default {
-  // no stylelint 16 support yet
-  // https://github.com/developer-stylechain/stylelint-a11y/issues/4
-  // plugins: ['stylelint-a11y'],
```

Comment:
do we know have access to the a11y rules?  otherwise, we can should leave this comment in as a reminder, or maybe its just time to make a formal issue for tracking.

Reply:
Oh, yeah I can revert this, or update to include our own issue, if you prefer. 

As for the support, it looks like the issue that you have linked above is (now) closed and has [a comment with a resolution](https://github.com/developer-stylechain/stylelint-a11y/issues/4#issuecomment-1927321522) of using 2 other libraries instead of the `developer-stylechain` project.

Styleline v15 -> https://www.npmjs.com/package/@ronilaukkarinen/stylelint-a11y
Styleline v16 -> https://www.npmjs.com/package/@double-great/stylelint-a11y

Did you want me to attempt to include that plugin here in this PR? Happy to try, and could spin off a new PR if things got messy, if that's cool.